xml/dhcp_vendor_class.xml

```diff
@@ -897,4 +897,18 @@
     <param pos="0" name="os.vendor" value="Cambium Networks"/>
   </fingerprint>
 
+  <fingerprint pattern="^LG_\d{2}[ULPS][A-Z]">
+    <description>LG TV</description>
```

Comment:
Would "LG Smart TV" be a more appropriate description? This would also match the existing favicon fingerprint.

Reply:
Yes, done, thanks.